ioctl/cmd/bc/bcbucketlist.go

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2020 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package bc
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/grpc-ecosystem/go-grpc-middleware/util/metautils"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc/status"
+
+	"github.com/iotexproject/iotex-proto/golang/iotexapi"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+)
+
+// Multi-language support
+var (
+	bcBucketListCmdShorts = map[config.Language]string{
+		config.English: "Get bucket list from block chain with given address",
+		config.Chinese: "",
+	}
+	bcBucketListUses = map[config.Language]string{
+		config.English: "bucketlist [ALIAS|ADDRESS]",
+		config.Chinese: "",
+	}
+)
+
+// bcBucketListCmd represents the bc bucketlist command
+var bcBucketListCmd = &cobra.Command{
+	Use:   config.TranslateInLang(bcBucketListUses, config.UILanguage),
+	Short: config.TranslateInLang(bcBucketListCmdShorts, config.UILanguage),
+	Args:  cobra.RangeArgs(0, 1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		arg := ""
+		if len(args) == 1 {
+			arg = args[0]
+		}
+		err := getBucketList(arg)
+		return output.PrintError(err)
+	},
+}
+
+type bucketlistMessage struct {
+	Node       string                     `json:"node"`
+	Bucketlist *iotextypes.VoteBucketList `json:"bucketlist"`
+}
+
+func (m *bucketlistMessage) String() string {
+	if output.Format == "" {
+		var lines []string
+		lines = append(lines, fmt.Sprintf("Blockchain Node: %s", m.Node))
+		for _, bucket := range m.Bucketlist.Buckets {
+			lines = append(lines, output.JSONString(bucket))
+		}
+		return strings.Join(lines, "\n")
+	}
+	return output.FormatString(output.Result, m)
+}
+
+// getBucketList get bucket list from chain
+func getBucketList(arg string) error {
+	address, err := util.GetAddress(arg)
+	if err != nil {
+		return output.NewError(output.AddressError, "", err)
+	}
+	bucketlist, err := getBucketListByAddress(address)
+	if err != nil {
+		return err
+	}
+	message := bucketlistMessage{Node: config.ReadConfig.Endpoint, Bucketlist: bucketlist}
+	fmt.Println(message.String())
```

Comment:
can you paste an example output?

Reply:
```Blockchain Node: api.testnet.iotex.one:443
{
  "index": 34,
  "candidateAddress": "io1t56twy23yjuqscljpjc869hyqw3gpswwj0g228",
  "stakedAmount": "100000000000000000000",
  "stakedDuration": 7,
  "createTime": {
    "seconds": 1589179280
  },
  "stakeStartTime": {
    "seconds": 1589179280
  },
  "unstakeStartTime": {},
  "owner": "io120au9ra0nffdle04jx2g5gccn6gq8qd4fy03l4"
}
{
  "index": 43,
  "candidateAddress": "io1l40dc4q95fsjdprhcga5qrpdsj46q3wpzr27n9",
  "stakedAmount": "170000000000000000000",
  "stakedDuration": 14,
  "createTime": {
    "seconds": 1589424135
  },
  "stakeStartTime": {
    "seconds": 1589424135
  },
  "unstakeStartTime": {},
  "owner": "io120au9ra0nffdle04jx2g5gccn6gq8qd4fy03l4"
}

Reply:
1. "stakedAmount": 100 IOTX
2. "stakeStartTime": use time.Format() to output time in string format
3. "stakedDuration": 7 days,
4. move "owner" to after "index"
5. "candidateAddress" --> "candidate"

Reply:
```
Blockchain Node: api.testnet.iotex.one:443
{
index: 34
owner: io120au9ra0nffdle04jx2g5gccn6gq8qd4fy03l4
candidate: io1t56twy23yjuqscljpjc869hyqw3gpswwj0g228
stakedAmount: 100 IOTX
stakedDuration: 7 days
createTime: 2020-05-11T06:41:20Z
stakeStartTime: 2020-05-11T06:41:20Z
unstakeStartTime: none
}
{
index: 43
owner: io120au9ra0nffdle04jx2g5gccn6gq8qd4fy03l4
candidate: io1l40dc4q95fsjdprhcga5qrpdsj46q3wpzr27n9
stakedAmount: 170 IOTX
stakedDuration: 14 days
createTime: 2020-05-14T02:42:15Z
stakeStartTime: 2020-05-14T02:42:15Z
unstakeStartTime: none
}